### DISCUSSION TITLE
Enable http3

### DISCUSSION THREAD

**Discussion by geraistec** [shinsenter/php#194] [Q&A]
How to enable http3 for the Laravel PHP image?

How to upgrade Nginx to version > 1.25?

**Comment by shinsenter** [comment]
Hello @geraistec, sorry for the late response (it seems that Github did not display your issue earlier).

Nginx is installed as the default version available from the Debian or Alpine Linux Docker images. As far as I know, the latest version is 1.26.2, and it does not come with the necessary module for HTTP/3 support.

I think you can use the `shinsenter/php:8.4-fpm` tag and manually install an Nginx version that supports HTTP/3 for your website.
https://hub.docker.com/r/shinsenter/php/tags?name=8.4-fpm&ordering=-name

**Reply by geraistec** [reply]
> Hello @geraistec, sorry for the late response (it seems that Github did not display your issue earlier).
> 
> Nginx is installed as the default version available from the Debian or Alpine Linux Docker images. As far as I know, the latest version is 1.26.2, and it does not come with the necessary module for HTTP/3 support.
> 
> I think you can use the `shinsenter/php:8.4-fpm` tag and manually install an Nginx version that supports HTTP/3 for your website. https://hub.docker.com/r/shinsenter/php/tags?name=8.4-fpm&ordering=-name

The GitHub account did have a problem, but it has been resolved.

As of version 1.25, the http3 module is now available natively.

https://nginx.org/en/docs/quic.html
